resources/com.yubico.yubioath.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.yubico.yubioath</id>
+  <launchable type="desktop-id">com.yubico.yubioath.desktop</launchable>
+
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
Need to double check the license choice a bit internally before merging, but is probably fine.

Reply:
Sure, no rush

Reply:
@dainnilsson any input here? https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html here are the supported licenses, under `metadata_license`

Reply:
If I'm reading this correctly, this license _only_ affects the metadata file itself? If so, I'd go with MIT as it's the one I'm most familiar with and it quite similar to the BSD license we use for the project.

Reply:
Agreed, MIT is my suggestion as well. 

@ai Can you update the PR to use MIT for the metadata_license ?

Reply:
Fixed